.editorconfig

```diff
@@ -0,0 +1,23 @@
+# The .editorconfig is used to maintain consistent code style.
+# The .editorconfig file is supported by most text editors.
+# See https://editorconfig.org
+
+root = true
+
+[*]
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
```

Comment:
Java should be 4, imo
Agreed on the rest.

Reply:
Huh, just reading that the Google style guide is 4 for Java, interesting. I knew that to be true of JS but Java that's new to me!

Reply:
The official Java (Oracle) guideline is 4 spaces: https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html

Reply:
4 for Java sounds good to me! :)
I think deviating from Google's Java style guide a little should be fine here (since Online Boutique's primary users aren't necessarily Googlers).